src/operations.cc

```diff
@@ -68,8 +67,8 @@ namespace sharp {
     // Extract luminance
     VImage luminance = lab[0];
     // Find luminance range
-    int const min = luminance.percent(1);
-    int const max = luminance.percent(99);
+    int const min = luminance.percent(lowerBin);
```

Comment:
Could optimise slightly here when the values are 0 or 100, which would be `min()` or `max()` respectively.

Reply:
[resolved](https://github.com/lovell/sharp/pull/3578/commits/f0cd634c86f5734d2a534b4becc594170e8195df)